### DISCUSSION TITLE
Is it appropriate to use HISAT2-aligned short-read data for SQANTI3 analysis?

### DISCUSSION THREAD

**Discussion by yycc9897** [ConesaLab/SQANTI3#360] [Q&A] [answered]
Hi,
I am currently using SQANTI3 for long-read transcriptome data analysis and have a question regarding the input data. According to the SQANTI3 documentation, the short-read BAM files should be generated by aligning short reads against the reference genome using a splice-aware mapper, such as STAR. However, I have used HISAT2 for alignment.
My question is: Is it appropriate to use HISAT2-aligned short-read data for SQANTI3 analysis? Will it affect the quality control and functional annotation results provided by SQANTI3?
Thank you for your help!

**Comment by carolinamonzo** [comment, marked as answer]
Hi @yycc9897, it's not a problem at all, you can use Hisat2 to map your short read data :)
